spec/system/users/edit_user_spec.rb

```diff
@@ -6,11 +6,14 @@
   let(:new_email) { Faker::Internet.email }
   let(:password) { Faker::Internet.password(min_length: 8) }
   let!(:user) { create(:user, password: password) }
+  let(:edit_user_page) { PageObjects::EditUser.new }
 
   feature 'being logged out' do
-    scenario 'can not access page' do
-      visit edit_user_registration_path
+    before do
+      edit_user_page
+    end
 
+    scenario 'can not access page' do
       expect(page).to have_text('You need to sign in or sign up before continuing')
```

Comment:
this expectation can be moved to the page object. Something like:
`expect(edit_user_page).to have_sign_in_required_message`

and then in the form object
```ruby
def has_sign_in_required_message?
  has_text?('You need to sign in or sign up before continuing')
end
```

Reply:
The `expect(edit_user_page).to have_sign_in_required_message` should be in the page object? And how do we call this from the spec?
Or we can do something like this on the spec `expect(edit_user_page.has_sign_in_required_message?).to be true`. 
 WDYT?

Reply:
no in the spec you call `expect(edit_user_page).to have_sign_in_required_message` and Capybara will magically call `has_sign_in_required_message?` on the page object (note how the conjugation of the method name changes)

Reply:
Awesome!! 🚀 🚀